README.md

```diff
@@ -28,6 +28,23 @@ await server.register({
 })
 ```
 
+## Plugin registration with Hemera injection
```

Comment:
I wouldn't name it `injection` rather `"Instantiate hemera outside"` or  `"Pass a custom hemera instance"`